examples/prism-factor.html

```diff
@@ -0,0 +1,86 @@
+<h2>Comments</h2>
+<pre> <code>! a comment</code> </pre>
```

Comment:
Please remove the extra spaces. The other one below as well.

```suggestion
<pre><code>! a comment</code></pre>
```